messages/messages.go

```diff
@@ -52,6 +52,7 @@ var (
 	HOUSTON_DEPLOYMENT_HEADER        = "Authenticated to %s \n\n"
 	HOUSTON_DEPLOYING_PROMPT         = "Deploying: %s\n"
 	HOUSTON_NO_DEPLOYMENTS_ERROR     = "No airflow deployments found"
+	HOUSTON_DEPLOYMENT_NAME_ERROR    = "Please specify correct deployment name"
```

Comment:
Maybe something like
"Please specify a deployment that exists at link/deployments" or
"Please specify one of the following deployments:
deploy1
deploy2
...
" if there are some or otherwise "Please create a deployment in Astronomer before deploying"?